conftest.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+from sqlalchemy.engine import reflection
+from sqlalchemy.schema import MetaData, Table, DropTable, ForeignKeyConstraint, DropConstraint
+
+from cal import app as cal_app
+from models import db as cal_db
+
+
+def drop_everything(db):
+    # From http://www.sqlalchemy.org/trac/wiki/UsageRecipes/DropEverything
+
+    conn = db.engine.connect()
+
+    # the transaction only applies if the DB supports
+    # transactional DDL, i.e. Postgresql, MS SQL Server
+    trans = conn.begin()
+
+    inspector = reflection.Inspector.from_engine(db.engine)
+
+    # gather all data first before dropping anything.
+    # some DBs lock after things have been dropped in
+    # a transaction.
+    metadata = MetaData()
+
+    tbs = []
+    all_fks = []
+
+    for table_name in inspector.get_table_names():
+        fks = []
+        for fk in inspector.get_foreign_keys(table_name):
+            if not fk['name']:
+                continue
+            fks.append(
+                ForeignKeyConstraint((), (), name=fk['name'])
+            )
+        t = Table(table_name, metadata, *fks)
+        tbs.append(t)
+        all_fks.extend(fks)
+
+    for fkc in all_fks:
+        conn.execute(DropConstraint(fkc))
+
+    for table in tbs:
+        conn.execute(DropTable(table))
+
+    trans.commit()
+
+
+@pytest.fixture(scope="session")
+def app(request):
```

Comment:
Will I need to load CalendarOptions too? I thought I wouldn't because it was already done in cal.py